flutter/lib/main.dart

```diff
@@ -0,0 +1,53 @@
+//Flutter packages
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+//Third party packages
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:provider/provider.dart';
+//My packages
+import '../providers/superformula_provider.dart';
+import '../screens/home_screen.dart';
```

Comment:
**question:** what are some of the trade offs between using relative vs package imports?

Reply:
The relative method is shorter and fast whereas using absolute import provides more clarity and is especially useful in complex project structures with the same multiple file names.